src/Html.php

```diff
@@ -3547,12 +3540,7 @@ public static function initEditorSystem(
         }, $content_css_paths));
         $skin_url = preg_replace('/^.*href="([^"]+)".*$/', '$1', self::css('css/standalone/tinymce_empty_skin', ['force_no_version' => true]));
 
-        // TODO: the recent changes to $skin_url above break tinyMCE's placeholders
-        // Reverted to the previous version here, but this should be fixed properly
-        $skin_url = $CFG_GLPI['root_doc'] . "/public/lib/tinymce/skins/ui/oxide";
-
```

Comment:
The underlying TODO has not been fixed, tinymce placeholder are still invisible without this code.

![image](https://github.com/user-attachments/assets/931b6929-660e-4d3d-b22a-31aaeb97983f)